apps/backend/src/routes/me/index.ts

```diff
@@ -62,7 +62,7 @@ router.post<any, ApiResponseBody<StravaUpload>>(
       });
     }
     const fileContent = req.file.buffer.toString();
-    const activityName = (req.query.name || 'dundring.com workout') as string;
+    const activityName = (req.query.name || 'Dundring workout') as string;
```

Comment:
```suggestion
    const activityName = (req.query.name || 'dundring workout') as string;
```

Reply:
or
```suggestion
    const activityName = (req.query.name || 'dundring') as string;
```

Reply:
@sivertschou Went for the `dundring` one. Added, Amended and rebased 🚀